### ISSUE TITLE
Any chance of updating these demos to be compatible with Elixir 1.0.4?

### ISSUE THREAD

**Issue by thestonefox** [sasa1977/exactor_demos#1]
I'm trying to learn how to use Poolboy but I'm using Elixir 1.0.4 and these demos don't seem to work for me.

I'd be really happy to just see an example of the:

```
exactor_demos/lib/parallel/parallelization_demo.ex
```

converted to work and run under Elixir 1.0.4

**Comment by sasa1977** [comment]
Yeah, these demos are two years old, and need to be updated. Unfortunately, I'm currently swamped with work, so I can't deal with this immediately. I'll keep it in mind, and do it as soon as I find some spare time, but I can't commit to any deadline.

**Comment by thestonefox** [comment]
Ok cool, thanks for the response :)

If I get my head around poolboy and get a version working with 1.0.4 then I may fork this repo and try updating them for you :)
